fdbclient/FDBTypes.h

```diff
@@ -1126,4 +1126,27 @@ inline const char* transactionPriorityToString(TransactionPriority priority, boo
 	throw internal_error();
 }
 
+struct VersionVector {
+	std::map<Tag, Version> versions;
+
+	VersionVector() {}
```

Comment:
The map is going to be empty when the sequencer starts. The semantics is that if we don't have an entry for a storage in the map then we don't know the commit version till which the storage has peeked -  the client wouldn't be able to provide the latest commit version to the storage (as part of the read request) and in that case we will require the storage to be up-to-date with the read version in order to answer the read request.